src/utils/projects.js

```diff
@@ -51,4 +149,11 @@ function treeIdGenerator(projectToImport) {
   ];
 }
 
-export { projectStatus, treeIdGenerator };
+export {
+  projectStatus,
+  getPaths,
+  getDocsInfos,
+  createTreeFromPaths,
+  matchFoldersAndDocs,
+  treeIdGenerator
```

Comment:
I think these new functions should be in the **files** utils instead of project utils.